multiversx_sdk_cli/delegation/staking_provider.py

```diff
@@ -102,7 +108,13 @@ def prepare_transaction_for_removing_nodes(self, owner: IAccount, args: Any) ->
 
     def prepare_transaction_for_staking_nodes(self, owner: IAccount, args: Any) -> ITransaction:
         delegation_contract = Address.new_from_bech32(args.delegation_contract)
-        public_keys = self._parse_public_bls_keys(args.bls_keys)
+
+        if args.bls_keys:
+            public_keys = self._parse_public_bls_keys(args.bls_keys)
+        else:
+            validators_file_path = Path(args.validators_file).expanduser()
+            validators_file = ValidatorsFile(validators_file_path)
+            public_keys = validators_file.load_public_keys()
```

Comment:
Can it be extracted to `_load_validator_public_keys(args)` (duplicated below)?

Reply:
Extracted the code in a method